segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -173,6 +192,23 @@ public void initChannel(SocketChannel ch) throws Exception {
              }
          });
 
+        if (enableTls && enableTlsReload) {
+            log.debug("TLS reload is enabled, so setting up a FileChangeWatcher object to watch changes in file: {}",
+                    this.pathToTlsCertFile);
+            try {
+                tlsCertFileChangeWatcherTask = new FileModificationWatcher(
+                        this.pathToTlsCertFile,
+                        new TLSConfigChangeEventConsumer(sslCtx, this.pathToTlsCertFile, this.pathToTlsKeyFile));
+                tlsCertFileChangeWatcherTask.setDaemon(true);
+
+                tlsCertFileChangeWatcherTask.start();
+                log.info("Done setting up TLS reload, which in turn will be based on changes in file: {}",
+                        this.pathToTlsCertFile);
+            } catch (FileNotFoundException e) {
+                log.warn("Failed to setup a watcher on the file {}", this.pathToTlsCertFile, e);
```

Comment:
we should change it to ERROR level since this would prevent the SSS service from starting.

Reply:
Fixed.